packages/react-components/src/components/chart/converters/convertSeriesAndYAxis.ts

```diff
@@ -1,11 +1,26 @@
 import { useMemo } from 'react';
-import { DataStream } from '@iot-app-kit/core';
+import { DataPoint, DataStream } from '@iot-app-kit/core';
 import { SeriesOption, YAXisComponentOption } from 'echarts';
 
+import maxBy from 'lodash.maxby';
+import minBy from 'lodash.minby';
+
 import { ChartAxisOptions, ChartStyleSettingsOptions } from '../types';
 import { convertDataPoint } from './convertDataPoint';
 import { StyleSettingsMap, getChartStyleSettingsFromMap } from './convertStyles';
 import { convertYAxis as convertChartYAxis } from './convertAxis';
+import { ChartStyleSettingsWithDefaults } from '../utils/getStyles';
+
+const EMPHASIZE_SCALE_CONSTANT = 2;
+const DEEMPHASIZE_OPACITY = 0.25;
+
+export type YAxisLegendOption = { datastream: DataStream; value: DataPoint; color: string; significantDigits: number };
```

Comment:
we can move all the types to the types.ts files ?  all the types are moved/created in types.ts in general